inspectit-oce-agent/src/system-test/java/rocks/inspectit/oce/metrics/GCMetricsSysTest.java

```diff
@@ -81,7 +81,7 @@ public void testGCPauseCapturing() throws Exception {
                 .findFirst().get();
 
 
-        assertThat(((AggregationData.SumDataLong) minorTime.getValue()).getSum()).isGreaterThan(0);
+        assertThat(((AggregationData.SumDataLong) minorTime.getValue()).getSum()).isGreaterThanOrEqualTo(0);
         assertThat(((AggregationData.SumDataLong) majorTime.getValue()).getSum()).isGreaterThan(0);
```

Comment:
Can it happen, that we had only minor GCs but not a major one, so this would be 0? Or is the notification only sent on major ones?

Reply:
I don't think we experienced this yet.. what I experienced here was that my minorgc was so fast that it had 0ms pause time.. that's why I adjusted the check